plugins/inputs/clickhouse/clickhouse_test.go

```diff
@@ -74,8 +74,9 @@ func TestGather(t *testing.T) {
 						},
 					},
 				})
+				assert.NoError(t, err)
```

Comment:
Now is a good chance to change all `assert` to `require` in this file ;)

Reply:
You can't... Check line 17... :-) I didn't want to rewrite the whole tests... This PR is a rabbit hole anyway. :-)

Reply:
I will leave this one as is. If you agree, please mark as resolved! :-)